crates/lib/src/executable.rs

```diff
@@ -502,13 +507,23 @@ impl<'execution> Executable<'_, 'execution> {
     /// Call [`Executable::retrieve_results`] to wait for execution to complete and retrieve the
     /// results.
     ///
+    /// # Arguments
+    /// 1. `quantum_processor_id`: The name of the QPU to translate the program for on.
```

Comment:
```suggestion
    /// 1. `quantum_processor_id`: The ID of the QPU for which to translate the program.
```

(recognizing that it was copied from above)